Mathlib/RingTheory/JacobsonIdeal.lean

```diff
@@ -214,6 +225,41 @@ theorem jacobson_mono {I J : Ideal R} : I ≤ J → I.jacobson ≤ J.jacobson :=
   erw [mem_sInf] at hx ⊢
   exact fun K ⟨hK, hK_max⟩ => hx ⟨Trans.trans h hK, hK_max⟩
 
+theorem jacobson_mul_mem_right {I : Ideal R}
+    (mul_mem_right : ∀ {x y}, x ∈ I → x * y ∈ I) :
```

Comment:
Please add a docstring pointing the user to `TwoSidedIdeal.jacobson` with a statement that they should prefer that when the ideal is two-sided.